pallets/anchors/src/lib.rs

```diff
@@ -306,7 +310,7 @@ pub mod pallet {
 				.ok_or(ArithmeticError::Underflow)?;
 
 			let fee = base_fee
-				.checked_mul(&pallet_fees::BalanceOf::<T>::from(multiplier))
+				.checked_mul(&pallet_fees::BalanceOf::<T>::from(multiplier as u8))
```

Comment:
Why is the as u8 needed now? I would like to use a `try_into().ok_or(ArithmethicError::Overflow)` here, as u8 is really small to cast to.

Reply:
Ah sure, well while testing it was complaining about the types. But indeed, I was thinking along the same lines as `u8` is really small to cast to.